ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/compact-3node-cluster/provision/cucushift-installer-rehearse-aws-ipi-compact-3node-cluster-provision-chain.yaml

```diff
@@ -0,0 +1,18 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-compact-3node-cluster-provision
+  steps:
+  - chain: ipi-conf-aws
+  - chain: ipi-install-install-aws
```

Comment:
There is no `ipi-install-install-aws` chain, please make sure the chain/step exits before you commit the code.